optimize_workspace.sh

```diff
@@ -1,19 +1,33 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
-command -v shellcheck > /dev/null && shellcheck "$0"
+command -v shellcheck >/dev/null && shellcheck "$0"
 
 export PATH="$PATH:/root/.cargo/bin"
 
 rustup toolchain list
 cargo --version
 
-optimize_workspace.py
+# Build artifacts
+echo -n "Building artifacts in workspace..."
+/usr/local/bin/build_workspace.py
```

Comment:
Ther idea here was that `/usr/local/bin/` is in `$PATH` so we can call the script by name, not by path. But this is fine as well.

Reply:
This is just standard practice for me, for custom / own scripts. Can revert it if you prefer.

Reply:
Either way is good. Just leave it like it is now.